recipes/highered/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "highered" %}
+{% set version = "0.2.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/highered-{{ version }}.tar.gz
+  sha256: 5fcae90599dda98560d6f5347ff88aee56a4b8e7181d009852c3dc699c336fb7
+
+build:
+  number: 0
+  skip: true  # [win]
```

Since this is pure Python, go ahead and make this noarch, since it drastically cuts down the number of builds:
```suggestion
  noarch: python
```

This will result in a build failure on Windows, but we can ignore it here. When the feedstock is made, noarch packages are only built on Linux anyway.

---

Perfect, all set then.